src/main/scad/alfons-eyckmans/aracna/eyckmans.aracna.scad

```diff
@@ -0,0 +1,55 @@
+/* ====================================================================
+
+  This is a 3D model of a mechanical puzzle. It is released under
+  the following license:
+
+  Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported
+  https://creativecommons.org/licenses/by-nc-nd/3.0/
+
+  This means the model is licensed for personal, noncommercial use
+  only. Anyone may print a copy for their own use, but selling or
+  otherwise monetizing the model or print (or any derivatives) is
+  prohibited. For details, refer to the summary at the above URL.
+
+  Puzzle design (c) Alfons Eyckmans
+  3D model (c) Tom Burns
+
+==================================================================== */
+
+include <puzzlecad.scad>
+
+require_puzzlecad_version("2.1");
+
+$burr_scale = 9;
+$burr_bevel = 1;
+$auto_layout = true;
+
+//print one copy each of spider() and sticks() below:
+
+*spider();
+*sticks();
+
+module spider() {
+  burr_piece(
+    [".x..x.|......|......|x....x|......|.x..x.",
+     ".x..x.|.xxxx.|..xx..|xxxxxx|.xxxx.|.x..x.",
+     "......|......|......|..xx..|..xx..|......",]
+  );
+}
+
+module sticks() {
+ burr_plate([
+    ["x......x|x.xx..xx|xxxxxxxx","x......x|x.....xx|xx..x.xx"],
+    ["x......x|x.xx...x|xxxxxxxx","x......x|x......x|xx....xx"],
+    ["x......x|x.x....x|xxxxxxxx","x......x|x......x|xx.xx.xx"],
+    ["x......x|x.x....x|xxxxxxxx","x......x|x......x|xx...xxx"],
+    ["x......x|x...xx.x|xxxxxxxx","x......x|x......x|xx....xx"],
+    ["x......x|x...xx.x|xxxxxxxx","x......x|x......x|xx....xx"],
+    ["x......x|x...x..x|xxxxxxxx","x......x|x......x|xx...xxx"],
+    ["x......x|x....xxx|xxxxxxxx","x......x|x.....xx|xx....xx"],
+    ["x......x|x....x.x|xxxxxxxx","x......x|x......x|xx...xxx"],
+    ["x......x|x....x.x|xxxxxxxx","x......x|x......x|xx....xx"],
+    ["x{label_text=Ara,label_orient=x-y-}......x|x......x|xxxxxxxx","x......x|x......x|xx.xx.xx"],
+    ["x......x{label_text=cna,label_orient=x+y+}|x......x|xxxx{label_text=Alfons Eyckmans,label_orient=y-x+,label_hoffset=0.5}xxxx","x......x|x......x|xx.xx.xx"],
```

Comment:
If you print the full text "Aracna" on both pieces with appropriate offsets, that will ensure that the characters line up properly when the puzzle is assembled. (Check out Save the Gorilla to see how I did this) Otherwise, the two halves of the text will be either too close or too far apart

Reply:
Fixed!